src/containers/widgets/constants.ts

```diff
@@ -1,113 +1,122 @@
     name: 'Mangrove Protection',
-    slug: 'mangrove_protection' satisfies WidgetSlugType,
+    slug: 'mangrove_protection',
     locationType: ['country', 'worldwide'],
     categoryIds: ['all_datasets', 'restoration_and_conservation'],
   },
   {
     name: 'Mangrove Restoration',
-    slug: 'mangrove_restoration' satisfies WidgetSlugType,
+    slug: 'mangrove_restoration',
     locationType: ['country', 'worldwide'],
     categoryIds: ['all_datasets', 'restoration_and_conservation', 'ecosystem_services'],
     layersIds: ['restoration'],
   },
   {
     name: 'Mangrove biomass',
-    slug: 'mangrove_biomass' satisfies WidgetSlugType,
+    slug: 'mangrove_biomass',
     locationType: ['custom-area', 'wdpa', 'country', 'worldwide'],
     categoryIds: ['all_datasets', 'restoration_and_conservation', 'ecosystem_services'],
     layersIds: ['biomass'],
   },
   {
     name: 'Mangrove height',
-    slug: 'mangrove_height' satisfies WidgetSlugType,
+    slug: 'mangrove_height',
     locationType: ['custom-area', 'wdpa', 'country', 'worldwide'],
     categoryIds: ['all_datasets', 'restoration_and_conservation', 'ecosystem_services'],
     layersIds: ['height'],
   },
   {
     name: 'Mangrove Blue Carbon',
-    slug: 'mangrove_blue_carbon' satisfies WidgetSlugType,
+    slug: 'mangrove_blue_carbon',
     locationType: ['custom-area', 'country', 'worldwide'],
     categoryIds: ['all_datasets', 'climate_and_policy', 'ecosystem_services'],
     layersIds: ['carbon'],
   },
   {
     name: 'Mangrove Emissions Mitigation',
-    slug: 'mangrove_emissions_mitigation' satisfies WidgetSlugType,
+    slug: 'mangrove_emissions_mitigation',
     locationType: ['custom', 'wdpa', 'country', 'worldwide'],
     categoryIds: ['all_datasets', 'climate_and_policy'],
   },
   {
     name: 'Mangrove International Status',
-    slug: 'mangrove_international_status' satisfies WidgetSlugType,
+    slug: 'mangrove_international_status',
     locationType: ['custom', 'wdpa', 'country'],
     categoryIds: ['all_datasets', 'climate_and_policy'],
   },
   {
     name: 'Carbon Market Potential',
-    slug: 'mangrove_carbon_market_potential' satisfies WidgetSlugType,
+    slug: 'mangrove_carbon_market_potential',
     locationType: ['country'],
     categoryIds: ['all_datasets', 'climate_and_policy'],
     layersIds: [],
   },
-  // {
-  //   name: 'Mangrove coverage',
-  //   slug: 'mangrove_coverage' satisfies WidgetSlugType,
-  //   locationType: [''],
-  //   categoryIds: ['all_datasets', 'distribution_and_change'],
-  //   layersIds: ['coverage-1996-2016'],
-  // },
+  {
+    name: 'Draw or upload an area',
+    slug: 'mangrove_drawing_tool',
+    locationType: ['custom-area', 'wdpa', 'country', 'worldwide'],
+    categoryIds: [],
+    layersIds: [],
+  },
 ] satisfies WidgetTypes[];
 
+export const ANALYSIS_WIDGETS_SLUGS: WidgetTypes['slug'][] = [
+  'mangrove_habitat_extent',
+  'mangrove_net_change',
+  'mangrove_height',
+  'mangrove_biomass',
+  'mangrove_blue_carbon',
```

Comment:
```suggestion
 //  'mangrove_blue_carbon',  TO - DO put back when API calculations get fixed
```

Reply:
As far as I know, the analysis for this widget is working, is that an issue?

Reply:
Yeah, there was an error with the calculations so the numbers we are getting from API are not correct, Alicia already have a ticket for it

Reply:
Yeah, there was an error with the calculations so the numbers we are getting from API are not correct, Alicia already have a ticket for it